apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManager.java

```diff
@@ -188,25 +214,30 @@ public synchronized void refreshFilters() {
      */
     try {
       // If new batches are present, then build new filters for the affected files
-      final Date currentLastDatabaseUpdate = fetchLastLoadedBatchTime();
-      if (this.transactionTime == null || this.transactionTime.before(currentLastDatabaseUpdate)) {
+      final Date currentLastBatchCreated =
+          fetchLastLoadedBatchCreated().orElse(BEFORE_LAST_UPDATED_FEATURE);
+      if (this.lastBatchCreated == null || this.lastBatchCreated.before(currentLastBatchCreated)) {
         LOGGER.info(
```

Comment:
If I'm not mistaken the parameters have changed but the general logic behind this statement has not correct?  I ask because currently in the test environment this is logging every second and causing logs to fill up quite a bit.

Reply:
This code shouldn't put a log entry every second, only when new RIF batch was loaded. It doesn't on my laptop. Can you send me the logs where it is?

Reply:
Got your logs. I understand the problem and was able to reproduce it locally. The cause was using now() for the case where there was an empty LoadedBatches table.  I also confirmed that this change fixes the problem.